aws_iam_account_password_policy.tf

```diff
@@ -4,40 +4,40 @@
 
 
 resource "aws_iam_account_password_policy" "passwordRequireMinLength" {
-  minimum_password_length        = 6
+  minimum_password_length = 14
 }
```

Comment:
**Accurics remediation**
 
**[MEDIUM]: Password Policy must require minimal length of 14**
 
**Vulnerability**
Setting a lengthy password increases account resiliency against brute force login attempts
 
**Remediation**
Password must be atleast 14 characters long to increase resiliency against brute force attack

source: accurics-